sling/core/pckgmgr/src/main/java/com/composum/sling/core/pckgmgr/PackageServlet.java

```diff
@@ -543,6 +564,55 @@ protected void installationDone(SlingHttpServletRequest request, SlingHttpServle
 
     }
 
+    protected class UninstallOperation implements ServletOperation {
+
+        @Override
+        public void doIt(SlingHttpServletRequest request, SlingHttpServletResponse response,
+            ResourceHandle resource)
+            throws RepositoryException, IOException {
+
+            JcrPackageManager manager = PackageUtil.createPackageManager(request);
+            JcrPackage jcrPackage = PackageUtil.getJcrPackage(manager, resource);
+
+            uninstallPackage(request, response, manager, jcrPackage);
+        }
+
+        protected void uninstallPackage(SlingHttpServletRequest request, SlingHttpServletResponse response,
```

Comment:
generally copy pasted "install" operation. common parts could be extracted if needed.